celery/backends/arangodb.py

```diff
@@ -210,3 +216,18 @@ def delete(self, key):
             logging.error(aql_err)
         except Exception as err:
             logging.error(err)
+
+    def cleanup(self):
```

Comment:
should this be documented?

Reply:
Done :-) I added the same documentation as the other backends that implement it.